lib/storage/bucket.js

```diff
@@ -864,9 +864,14 @@ Bucket.prototype.setMetadata = function(metadata, callback) {
 };
 
 /**
- * Upload a file to the bucket. This is a convenience method that wraps the
- * functionality provided by a File object, {module:storage/file}.
+ * Upload a file to the bucket. This is a convenience method that wraps
+ * {module:storage/file#createWriteStream}.
  *
+ * Resumable uploads are automatically enabled if your input file is larger than
+ * 5 MB. You can override this logic by specifying `options.resumable` as `true`
```

Comment:
If resumable uploads are enabled by default, do we need to say that they can override the options by specifying `true`? Would it be better to just say something like, "You can disable this logic by specifying `options.resumable` as `false`."?